rules/sinks/third_parties/sdk/facebook/javascript.yaml

```diff
@@ -18,5 +18,5 @@ sinks:
     domains:
       - "facebook.com"
     patterns:
-      - "((window|w|this)[.])?fbq[(].*"
+      - "(?i)(.*[.])?fbq[(].*"
```

Comment:
@ankit-privado can you double check if this sort of change from a specific prefix of `window|w|this` to any prefix introduces false-positives for all the trackers that you've added ? I am generally concerned about the regex catching other things.  Not just for this case but for all the regex patterns where we have `.*` as a prefix.

Reply:
Yes agree. It might introduce false positives, especially with regex having a smaller fixed match string. However, this prefix could be anything like obj.fbq("something"), and in some places this prefix might be absent as in this [example](https://github.com/rudderlabs/rudder-sdk-js/blob/4ad61db9408f37449d62368d192b5d06e36e87ca/packages/analytics-js-integrations/src/integrations/Posthog/browser.js#L116). We tried to specify the code rule as much as possible.